src/chrome/content/rules/Wikimedia.xml

```diff
@@ -72,25 +87,26 @@
 	<target host="wikilov.es" />
 	<target host="www.wikilov.es" />
 
-	<target host="wikimedia.org.il" />
-	<target host="www.wikimedia.org.il" />
-
 	<target host="wikimedia.or.id" />
 	<target host="www.wikimedia.or.id" />
+	
+	<target host="wikimedia.org.au" />
+	<!-- All subdomains *.wikimedia.org.au use an invalid security certificate. -->
+
+	<target host="wikimedia.org.il" />
+	<target host="www.wikimedia.org.il" />
 	<!--
-		blog.wikimedia.org.id uses an invalid security certificate.
+		blog.wikimedia.org.il uses an invalid security certificate.
 		The certificate is only valid for www.wikimedia.org.il and wikimedia.org.il.
 	-->
-
-	<target host="wikimedia.ch" />
-	<target host="*.wikimedia.ch" />
-		<test url="http://www.wikimedia.ch/" />
-		<test url="http://glam.wikimedia.ch/" />
-		<test url="http://intern.wikimedia.ch/" />
-		<test url="http://members.wikimedia.ch/" />
-		<test url="http://portal.wikimedia.ch/" />
-		<test url="http://test.wikimedia.ch/" />
-		<test url="http://upload-www.wikimedia.ch/" />
+	
+	<target host="wikimedia.org.uk" />
+	<target host="blog.wikimedia.org.uk" />
+	<target host="donate.wikimedia.org.uk" />
+	<!-- lists.wikimedia.org.uk does not support HTTPS -->
```

Comment:
same for:
qrpedia.wikimedia.org.uk
en.qrwp.wikimedia.org.uk

WFM:
qrwp.wikimedia.org.uk